### DISCUSSION TITLE
No response after Uploading document

### DISCUSSION THREAD

**Discussion by mhkasif** [community/community#68274] [GitHub Education]
### Hi everyone,

When I upload my document, it starts processing and then comes back on the same page with no error and no response.
what should I do?

**Comment by josefdc** [comment]
Hi @mhkasif 

When you're having issues uploading a document without any error message or response, consider these common factors:

**Internet Connection:** Check for a stable connection.
**File Size:** Ensure it's within the platform's allowed limit.
**File Format:** Verify the platform supports your file type.
**Web Browser**: Try a different one or update the current one.
**Cookies & Cache:** Clear them and retry.
**Server Issues:** The platform's server might be down.
**Browser Extensions**: They can sometimes interfere; disable and see.
**Compatibility:** Update outdated systems or browsers.
**Account Limitations:** Ensure you haven't exceeded any upload limits.

**Comment by mhkasif** [comment]
Tried another browser that has no extension, but still the same result. I have been trying for weeks but getting the same thing.

**Comment by samus-aran** [comment]
Hi @mhkasif 👋 

Are you able to take a screenshot of the issue? If you are also familiar with your browser's developer tools are there any console errors you can take a screenshot to help us trouble shoot?

**Comment by mhkasif** [comment]
https://drive.google.com/file/d/1-Ukps6uL4GE_fsnCVN1mu0S9voREx9qR/view?usp=sharing
Here is the screen recorded video where you can see the logs in console and problem I am facing.
